Content.Server/Dragon/Components/DragonRiftComponent.cs

```diff
@@ -37,4 +37,15 @@ public sealed partial class DragonRiftComponent : SharedDragonRiftComponent
 
     [ViewVariables(VVAccess.ReadWrite), DataField("spawn", customTypeSerializer: typeof(PrototypeIdSerializer<EntityPrototype>))]
     public string SpawnPrototype = "MobCarpDragon";
+
+    // Sunrise-Start
+    [DataField]
+    public int MaxAliveCarps = 16;
+
+    [ViewVariables(VVAccess.ReadOnly)]
+    public int AliveCarps = 0;
```

Comment:
по умолчанию должно быть ноль, то есть можно написать 
```suggestion
    [ViewVariables(VVAccess.ReadOnly)]
    public int AliveCarps;
```